Deployment/complete-solution.json

```diff
@@ -561,6 +564,7 @@
             "FUNCTIONS_EXTENSION_VERSION": "~1",
             "ServicebusQueueConnectionString": "[parameters('servicebusManageConnectionString')]",
```

Comment:
Shouldn't this be 'ServicebusEventImagesQueueConnectionString'?

Reply:
Yes, that would be the most obvious indeed.
However, there is a small nuance between the two. The one used in the web application needs to have the queue specified in the connectionstring. The one in the Function doesn't need this as it's specified in the attribute itself.
Quite confusing, I'll check if I can fix some of this in the template, or readme.

Reply:
For completeness, the other comment was written on my phone, the connectionstring in the Function looks like this `Endpoint=sb://[myServicebus].servicebus.windows.net/;SharedAccessKeyName=FunctionsAccessKey;SharedAccessKey=[theMainPrimaryKey]`

The connectionstring in the App Service looks like this: `Endpoint=sb://[myServicebus].servicebus.windows.net/;SharedAccessKeyName=SendEventImages;SharedAccessKey=[aDifferentAccessKeyDueToTargettingADifferentEntityInAzure];EntityPath=eventimages`

Hope this clears it up a bit :-)

Reply:
Check! Those were already there! Now I just have to get #89 right and I can start deploying again